core/src/prediction/Prediction.cpp

```diff
@@ -20,14 +20,25 @@
 Prediction::Prediction(const std::vector<double>& predictions):
   predictions(predictions),
   variance_estimates(0),
-  error_estimates(0) {}
+  error_estimates(0),
+  excess_error_estimates(0) {}
 
 Prediction::Prediction(const std::vector<double>& predictions,
```

Comment:
For simplicity, I think we should nuke this constructor (if we're providing debiased error estimates, we should always be returning the bias used to calculate them as well).